plugin/adsk/plugin/plugin.cpp

```diff
@@ -56,6 +56,7 @@
 #include <ufe/runTimeMgr.h>
 
 #include <basePxrUsdPreviewSurface/usdPreviewSurfacePlugin.h>
+#include <lookdevXUsd/LookdevXUsd.h>
```

Comment:
This line and the two calls below need to be wrapped in `#ifdef` and only compiled in when we built LookdevXUsd.